### ISSUE TITLE
MN missing from manifest during run

### ISSUE THREAD

**Issue by digimer** [ClusterLabs/anvil#440]
When running a manifest where MN1 was added, the MN is not shown.

**Comment by digimer** [comment]
This is a problem with Striker->generate_manifest()

```
Aug 17 12:19:13 an-striker01.alteeve.com node[3056777]:   "script": "918e504e-43aa-4cd5-8c9d-d0e1526e526f x Striker->generate_manifest \"{\\\"dns\\\":\\\"8.8.8.8,8.8.4.4\\\",\\\"domain\\\":\\\"alteeve.com\\\",\\\"manifest_uuid\\\":\\\"new\\\",\\\"mtu\\\":0,\\\"ntp\\\":\\\"\\\",\\\"prefix\\\":\\\"an\\\",\\\"sequence\\\":1,\\\"bcn_count\\\":1,\\\"sn_count\\\":1,\\\"ifn_count\\\":1,\\\"mn_count\\\":1,\\\"bcn1_gateway\\\":\\\"\\\",\\\"bcn1_network\\\":\\\"10.201.0.0\\\",\\\"bcn1_subnet\\\":\\\"255.255.0.0\\\",\\\"sn1_gateway\\\":\\\"\\\",\\\"sn1_network\\\":\\\"10.101.0.0\\\",\\\"sn1_subnet\\\":\\\"255.255.0.0\\\",\\\"ifn1_gateway\\\":\\\"10.255.255.254\\\",\\\"ifn1_network\\\":\\\"10.255.0.0\\\",\\\"ifn1_subnet\\\":\\\"255.255.0.0\\\",\\\"mn1_gateway\\\":\\\"\\\",\\\"mn1_network\\\":\\\"10.199.0.0\\\",\\\"mn1_subnet\\\":\\\"255.255.0.0\\\",\\\"node1_ipmi_ip\\\":\\\"10.201.11.1\\\",\\\"node1_bcn1_ip\\\":\\\"10.201.10.1\\\",\\\"node1_sn1_ip\\\":\\\"10.101.10.1\\\",\\\"node1_ifn1_ip\\\":\\\"10.255.10.1\\\",\\\"node1_mn1_ip\\\":\\\"10.199.10.1\\\",\\\"node1_fence_an-pdu01\\\":\\\"1\\\",\\\"node1_fence_an-pdu02\\\":\\\"1\\\",\\\"node1_ups_an-ups01\\\":\\\"checked\\\",\\\"node1_ups_an-ups02\\\":\\\"checked\\\",\\\"node2_ipmi_ip\\\":\\\"10.201.11.2\\\",\\\"node2_bcn1_ip\\\":\\\"10.201.10.2\\\",\\\"node2_sn1_ip\\\":\\\"10.101.10.2\\\",\\\"node2_ifn1_ip\\\":\\\"10.255.10.2\\\",\\\"node2_mn1_ip\\\":\\\"10.199.10.2\\\",\\\"node2_fence_an-pdu01\\\":\\\"2\\\",\\\"node2_fence_an-pdu02\\\":\\\"2\\\",\\\"node2_ups_an-ups01\\\":\\\"checked\\\",\\\"node2_ups_an-ups02\\\":\\\"checked\\\"}\"\n"
```

**Comment by digimer** [comment]
Confirmed that the MN is displayed properly in the UI. This issue was on the back-end only. Resolved in f23768a.
